### PR TITLE
SAR-7833: Update registration logic to support multiple entities

### PR DESCRIPTION
**Pull request by r-melvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +14,22 @@ This is a Scala/Play frontend to allow Partnerships to provide their information
    `sbt 'run 9722 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`
 
 ### Testing
+
```
Comment (kabuumu) [inline README.md:17]: Were all these formatting changes intentional on the readme?

File: TestREADME.md [hunk 2]
```diff
@@ -4,18 +4,20 @@
...
 2. Partnership Identification
 
     - Use stub for Partnership Known Facts SAUTR call
 
 #### GET test-only/general-partnership/create-journey
 
 ---
-This is a test entry point which simulates a service by triggering the initial POST call to set up a journey for General Partnerships.
+This is a test entry point which simulates a service by triggering the initial POST call to set up a journey for General
+Partnerships.
```
Comment (kabuumu) [inline TestREADME.md:20]: And again on the formatting. Looks a bit odd splitting the line here.

File: app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/views/test_create_journey.scala.html [hunk 3]
```diff
@@ -41,53 +41,53 @@
 }
 
 @html = {
-    @formWithCSRF(action = formAction) {
+@formWithCSRF(action = formAction) {
```
Comment (kabuumu) [inline app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/views/test_create_journey.scala.html:44]: Your auto format has removed an indent here

File: app/uk/gov/hmrc/partnershipidentificationfrontend/views/check_your_answers_page.scala.html [hunk 4]
```diff
@@ -82,8 +81,8 @@
                     )))
                 )
             )
-        }
-    )
+    }
+)
```
Comment (kabuumu) [inline app/uk/gov/hmrc/partnershipidentificationfrontend/views/check_your_answers_page.scala.html:85]: Might be worth looking at this formatting manually

**Review by kabuumu — COMMENTED** [review]
Might want to remove the auto format across everything, it seems to have caused a few problems.

File: README.md (hunk 1 above)
Comment (r-melvin) [reply README.md:17]: No they weren't, they're the result of auto formatting

**Pull request merged**